### PR TITLE
Make it beautiful!

### PR DESCRIPTION
**Pull request by sky-xo**
- Start refactor :-)

### CODE REVIEW

File: cat/js_css/cat_keys.js [hunk 1]
```diff
@@ -7,9 +7,13 @@
 // Key Bindings _____________________________________________
 
 document.onkeydown = function() {  //   KEY DOWN
+  let keyDirections = {'a': 'left', 'd': 'right'}
```
Comment (sky-xo) [inline cat/js_css/cat_keys.js:10]: Keep filling this thing out!

Comment (EagleDog) [reply cat/js_css/cat_keys.js:10]: Excellent suggestions! Thanks for the contribution!

**Review by EagleDog — APPROVED** [review]
This is extremely useful. Thank you for the help.

**Comment by EagleDog** [comment]
Oh glorious beautiful wonderful excellence of quality!

**Pull request merged**